### PR TITLE
Feature: Implement support for HEAD requests

### PR DESCRIPTION
**Pull request by aleksipakkala**

### CODE REVIEW

File: services/contentServices.go [hunk 1]
```diff
@@ -55,3 +58,25 @@ func (p *contentService) GetContent(response http.ResponseWriter, bucket string,
...
+	}
+
+	objectHandle := client.Bucket(bucket).Object(object)
+	if objectHandle != nil {
+		attrs, _ := objectHandle.Attrs(ctx)
+		fmt.Printf("%+v", attrs.Size)
+		response.Header().Set("Content-Length", strconv.FormatInt(attrs.Size, 10))
+		response.Header().Set("Content-Type", attrs.ContentType)
+	}
+	return nil
```
Comment (manishasharma11) [inline services/contentServices.go:81]: may be put a relevant message that file does not exist

**Review by manishasharma11 — APPROVED** [review]
sounds promising . just a minor comment :)

**Comment by aleksipakkala** [comment]
Can you @manishasharma11 check once more, was this what you meant with adding a message if the file doesn't exist?

File: services/contentServices.go (hunk 1 above)
Comment (manishasharma11) [reply services/contentServices.go:84]: please put the message in response

File: services/contentServices.go [hunk 2]
```diff
@@ -52,6 +55,31 @@ func (p *contentService) GetContent(response http.ResponseWriter, bucket string,
...
+		return fmt.Errorf("storage.NewClient: %w", err)
+	}
+	defer client.Close()
+
+	if err != nil {
+		return fmt.Errorf("os.Create: %w", err)
+	}
+
+	objectHandle := client.Bucket(bucket).Object(object)
+	if objectHandle != nil {
```
Comment (manishasharma11) [inline services/contentServices.go:76]: here we are checking if the object exist , if it doesn't then we need to inform in response a 404 status code saying that file was not found on gcs. This way you have a relevant message in the response that file doesn't exist

File: services/contentServices.go (hunk 1 above)
Comment (manishasharma11) [reply services/contentServices.go:91]: looks good

**Pull request merged**